### ISSUE TITLE
Release of dataset

### ISSUE THREAD

**Issue by RussRobin** [NorthSummer/ProximityQA#1]
Hi, thank you for your great work.

I wonder what is the scheduled time to release Proximity-110k dataset, as proposed in your arxiv paper?

Also, for QA stage 1 (What is the relative depth value of ...? ). What does the depth value in answer indicate. Is it the average relative depth for the object mask, or the bounding box?

Great thanks in advance!

**Comment by NorthSummer** [comment]
Hi, thank you for your interest. I have updated a link to the training data in the readme, hoping it will be helpful. We will release more detailed information of this work in 3 weeks, including evaluation data, evaluation code, model parameters, etc. BTW, the relative depth value is located by the center of bounding box annotations, we has been introduced in our paper.

**Comment by RussRobin** [comment]
Great thanks for your update! ill closet the issue.

**Comment by RussRobin** [comment]
Hi, sorry for asking again.
Is it possible for you to release the ground truth depth map used in the dataset for further use?
Many thanks.

**Comment by NorthSummer** [comment]
Using [MiDAS](https://github.com/isl-org/MiDaS) to generate depth maps is really convenient. We have no plans to release the depth maps because the dataset is too large (there are totally 110k depth maps).

**Comment by RussRobin** [comment]
Great thanks!
